### PR TITLE
std.builtin: make enum fields lowercase

### PR DESCRIPTION
**Pull request by RossComputerGuy**
This PR makes the container layout fields all lowercase, this is done as a stylization unification.

### CODE REVIEW

File: src/AstGen.zig [hunk 1]
```diff
@@ -5370,20 +5370,20 @@ fn containerDecl(
     switch (token_tags[container_decl.ast.main_token]) {
         .keyword_struct => {
             const layout = if (container_decl.layout_token) |t| switch (token_tags[t]) {
-                .keyword_packed => std.builtin.Type.ContainerLayout.Packed,
-                .keyword_extern => std.builtin.Type.ContainerLayout.Extern,
+                .keyword_packed => std.builtin.Type.ContainerLayout.@"packed",
+                .keyword_extern => std.builtin.Type.ContainerLayout.@"extern",
                 else => unreachable,
-            } else std.builtin.Type.ContainerLayout.Auto;
+            } else std.builtin.Type.ContainerLayout.auto;
```
Comment (InKryption) [inline src/AstGen.zig:5377]: Might be worth changing this to use inference while you're at it (`const layout: std.builtin.Type.ContainerLayout = ...`, `=> .@"packed"`, `=> .@"extern"`, `else .Auto`).

Comment (RossComputerGuy) [reply src/AstGen.zig:5377]: Yeah 👍. This PR is still very much WIP at the moment as you can see by the breaking CI and draft status. Hope to finish this tonight though.

**Comment by RossComputerGuy** [comment]
@nektro Eh, maybe but is used `@""` elsewhere and I already have gone through the effort.

File: src/AstGen.zig (hunk 1 above)
Comment (RossComputerGuy) [reply src/AstGen.zig:5376]: Done

**Comment by RossComputerGuy** [comment]
Passes CI now 🎉

**Review by Vexu — APPROVED** [review]

**Comment by RossComputerGuy** [comment]
@Vexu as this PR is approved and its been some time but this hasn't been merged, I'm curious as to when this PR could be merged? I hope it isn't too long from now since this PR does break every few days with the changes being made upstream.

**Comment by Vexu** [comment]
I'll merge it next time you get the CI green.

**Comment by RossComputerGuy** [comment]
@Vexu Fixed, CI is going to pass soon.

**Pull request merged**